README.md

```diff
@@ -1,10 +1,31 @@
 # purple-pluto
 
-Todo: add description
+## Description
+
+Tool for tracking visitor activity on your web application. Deploying the code from this repository will allow you to record and store data of (x,y) coordinates from visitor's following actions:
+
+- Mouse hoover
```

Comment:
although I think this would be a really cool event to track, I believe there is a typo here 😅